Plant/SUBSTOR-Potato/PT_BTHTIME.for

```diff
@@ -88,45 +96,10 @@
 
 !*---daily thermal unit for phenology
       TDU  = TT
+      SDU  = ST
 
 !*---daily average of temperature effect on maintenance respiration
       ETRM = ETR
-      !RETURN
```

Comment:
There is a principle in programming called DRY (Don't Repeat Yourself) which essentially means avoid adding unnecessary code and reuse as much as possible.